### PR TITLE
flet-client-flutter: 0.24.1 -> 0.25.2

### PR DESCRIPTION
**Pull request by lucasew**
- **flet-client-flutter: 0.24.1 -> 0.25.2**

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow started

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12519129837
**Extra args**: ``

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow finished

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12519129837


## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368599`

---
### `x86_64-linux`
<details>
  <summary>:x: 12 packages failed to build:</summary>
  <ul>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet-core</li>
    <li>python312Packages.flet-core.dist</li>
    <li>python312Packages.flet-runtime</li>
    <li>python312Packages.flet-runtime.dist</li>
    <li>python312Packages.flet.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet-core</li>
    <li>python313Packages.flet-core.dist</li>
    <li>python313Packages.flet-runtime</li>
    <li>python313Packages.flet-runtime.dist</li>
    <li>python313Packages.flet.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
  </ul>
</details>

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow started

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12520380100
**Extra args**: ``

**Comment by lucasew** [comment]
I'll reorganize the commit history later

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow finished

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12520380100


## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368599`

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet.dist</li>
  </ul>
</details>

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368599`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet.dist</li>
  </ul>
</details>

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368599`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.flet.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.flet.dist</li>
  </ul>
</details>

**Comment by lucasew** [comment]
Is the flutter builder supposed to build on Darwin?

File: pkgs/by-name/fl/flet-client-flutter/package.nix [hunk 1]
```diff
@@ -14,17 +14,18 @@
 , libplacebo
 , _experimental-update-script-combinators
 , flet-client-flutter
+, fletTarget ? "linux"
 }:
 
 flutter.buildFlutterApplication rec {
```
[the remark this answers is not in the document]
Comment (lucasew) [reply pkgs/by-name/fl/flet-client-flutter/package.nix]: done

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow started

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12526774039
**Extra args**: ``

**Comment by lucasewd** [comment]
# [nixpkgs-reviewd](https://github.com/nixosbrasil/nixpkgs-reviewd) workflow finished

**Status and logs**: https://github.com/nixosbrasil/nixpkgs-reviewd/actions/runs/12526774039


## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368599`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>python312Packages.flet</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.flet.dist</li>
    <li>python313Packages.flet</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.flet.dist</li>
  </ul>
</details>

**Review by ghost — APPROVED** [review]

**Review by GaetanLepage — CHANGES_REQUESTED** [review]

File: pkgs/by-name/fl/flet-client-flutter/package.nix [hunk 2]
```diff
@@ -14,17 +14,18 @@
...
-flutter.buildFlutterApplication rec {
+flutter324.buildFlutterApplication rec {
   pname = "flet-client-flutter";
-  version = "0.24.1";
+  version = "0.25.2";
 
   src = fetchFromGitHub {
     owner = "flet-dev";
     repo = "flet";
     rev = "v${version}";
```
[the remark this answers is not in the document]
Comment (lucasew) [reply pkgs/by-name/fl/flet-client-flutter/package.nix]: Done

**Review by GaetanLepage — APPROVED** [review]
LGTM

**Pull request merged**